network-api/networkapi/wagtailpages/models.py

```diff
@@ -475,6 +486,7 @@ class Homepage(MetadataPageMixin, Page):
         'NewsPage',
         'ParticipatePage',
         'MiniSiteNameSpace',
+        'RedirectingPage',
```

Comment:
Not sure about this one: should the root page be allow to have redirecting pages @alanmoo? It kind of feels like none of the homepage children should ever be "a different website", but that's not my call.

Reply:
Feels weird, but let it. The Internet Health Report was already like that for a while, except that it was hard coded into the primary navigation.